main.go

```diff
@@ -114,6 +115,8 @@ func main() {
 				awsManager.CleanEbsSnapshots(assumedRoleClient)
 				awsManager.CleanEbsVolumes(assumedRoleClient)
 			}
+			// After cleaning up every region we set the account state to Ready
+			awsv1alpha1.ResetAccountState(cli, account)
```

Comment:
This should only happen if there were no errors from any of the Clean functions

Reply:
Agreed